graphql/util_test.go

```diff
@@ -351,3 +353,63 @@ func TestConcurrentlyBuildVersionsMatchingTasksMap(t *testing.T) {
 	assert.Equal(t, versionsMatchingTasksMap["v3"], true)
 
 }
+
+func TestHasAnnotationPermission(t *testing.T) {
+	for tName, tCase := range map[string]func(ctx context.Context, t *testing.T){
+		"HasAnnotationPermission": func(ctx context.Context, t *testing.T) {
```

Comment:
I found myself getting a bit confused by the tests because I forgot that the new function handles View & Modify permissions. 😅 It might be easier to understand if we split the tests up into 2 larger cases: `"ViewAnnotationPermissions"` / `"ModifyAnnotationPermissions"`